public/typhoonQuiz/typhoonQuiz.js

```diff
@@ -26,9 +26,16 @@ window.onload=function(){
 
         if(gameState==1){
             alert("ゲームオーバー");
+            window.location.href="../ending/gameover.html?dist="+quizDestination;
         }
         else if(gameState==2){
             alert("ゴール");
+            window.location.href="../ending/gameclear.html?dist="+quizDestination;
+        }
+        else if(gameState==3){
+            alert("ゲームオーバー");
+            quizDestination="e-10"
+            window.location.href="../ending/gameover.html?dist="+quizDestination;
```

Comment:
```suggestion
            window.location.href="../ending/gameclear.html?dest="+quizDestination;
        }
        else if(gameState==3){
            alert("ゲームオーバー");
            quizDestination="e-10"
            window.location.href="../ending/gameover.html?dest="+quizDestination;
```

destinationの略ならdestが正しそうです。